### PR TITLE
Prepare release spectrum-1.7.0

### PR DESCRIPTION
**Pull request by yiyang00171**
Let's update it to 1.7.0

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,16 +11,16 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>spectrum</artifactId>
-    <version>1.6.1-SNAPSHOT</version>
+    <version>1.7.0</version>
```
Comment (jamesward) [inline pom.xml]: This needs too be `1.7.0-SNAPSHOT` since Maven does the actual release.

File: pom.xml [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
     <name>Spectrum</name>
     <description>WebJar for Spectrum</description>
     <url>http://webjars.org</url>
 
     <developers>
         <developer>
-            <id>jamesward</id>
-            <name>James Ward</name>
-            <email>james@jamesward.com</email>
+            <id>yiyang00171</id>
```
Comment (jamesward) [inline pom.xml]: You can just add a new `<developer>` block.  You don't need to remove the existing one.

File: pom.xml [hunk 3]
```diff
@@ -36,12 +36,12 @@
         <url>http://github.com/webjars/spectrum</url>
         <connection>scm:git:https://github.com/webjars/spectrum.git</connection>
         <developerConnection>scm:git:https://github.com/webjars/spectrum.git</developerConnection>
-        <tag>HEAD</tag>
+        <tag>spectrum-1.7.0</tag>
```
Comment (jamesward) [inline pom.xml]: This should be `HEAD`.

File: pom.xml (hunk 1 above)
Comment (yiyang00171) [inline pom.xml]: `SNAPSHOT` has been added in the version.

File: pom.xml (hunk 2 above)
Comment (yiyang00171) [inline pom.xml]: Sorry :-) I've restored your developer block.

File: pom.xml (hunk 3 above)
Comment (yiyang00171) [inline pom.xml]: `tag` has been changed to `HEAD`

**Pull request merged**